### ISSUE TITLE
Dropping a distributed table drops shards from worker no longer in the cluster

### ISSUE THREAD

**Issue by mtuncer** [citusdata/citus#956]
1. Created a distributed table,
2. Removed a worker from pg_dist_node by deleting the row
3. Drop the distributed table

Result:
Shards in the removed worker are stills dropped even though the worker is not in the cluster anymore.

Expected:
Not that. Error/warning message perhaps.

Maybe we should have some sort of trigger to prevent manual changes in pg_dist_node.

**Comment by metdos** [comment]
> Removed a worker from pg_dist_node by deleting the row

I really think that the user shouldn't mess up with the metadata like this.

**Comment by ozgune** [comment]
@metdos / @mtuncer -- I agree that the user shouldn't manually delete a row from ``pg_dist_node``.

At the same time, we used to provide functionality pre-Citus 6.0 where the user could temporarily disable a worker node. For example, this could happen if the user was using append distribution and one of their worker nodes temporarily failed. To avoid spurious warning messages on ``SELECT``s, they would remove the worker node from ``pg_worker_list.conf`` and send a ``SIGHUP`` to Citus. This way, Citus wouldn't route any further queries to the worker node.

Once the node came back up, the user would add back the node to the membership list and send another ``SIGHUP``.

We talked about this functionality and temporarily disabling a node in #931. We haven't yet reached out a conclusion on it.
